web/src/main/webapp/xslt/services/pdf/metadata-fop.xsl

```diff
@@ -219,244 +205,197 @@
-        <fo:block>
-          <xsl:if test="$block != ''">
-            <xsl:if test="$label!=''">
-              <fo:inline font-size="{$title-size}" font-weight="{$title-weight}"
-                         color="{$title-color}" margin="8pt">
-                <xsl:value-of select="$label"/>
-              </fo:inline>
-            </xsl:if>
-            <fo:table width="100%" table-layout="fixed">
-              <fo:table-column column-width="5cm"/>
-              <fo:table-column column-width="15cm"/>
-              <fo:table-body>
-                <xsl:copy-of select="$block"/>
-              </fo:table-body>
-            </fo:table>
+                      <xsl:call-template name="metadata-resources">
+                        <xsl:with-param name="gui" select="$oldGuiStrings"/>
+                        <xsl:with-param name="metadata" select="."/>
+                      </xsl:call-template>
+
+                      <xsl:call-template name="info-rows">
+                        <xsl:with-param name="label" select="$oldGuiStrings/onTheWeb"/>
+                        <xsl:with-param name="content">
+                          <fo:inline>
+                            <xsl:value-of select="$oldGuiStrings/show"/>
+                            <xsl:text> </xsl:text>
+                            <fo:basic-link text-decoration="underline" color="$link-color">
+                              <xsl:attribute name="external-destination">url('<xsl:value-of
+                                select="concat($nodeUrl, 'api/records/', geonet:info/uuid)"
+                              />')
+                              </xsl:attribute>HTML
+                            </fo:basic-link>
+                            <fo:basic-link text-decoration="underline" color="$link-color">
+                              <xsl:attribute name="external-destination">url('<xsl:value-of
+                                select="concat($nodeUrl, 'api/records/', geonet:info/uuid, '/formatters/xsl-view?view=advanced&amp;output=pdf')"/>')
+                              </xsl:attribute>PDF
+                            </fo:basic-link>
+                            <fo:basic-link text-decoration="underline" color="$link-color">
+                              <xsl:attribute name="external-destination">url('<xsl:value-of
+                                select="concat($nodeUrl, 'api/records/', geonet:info/uuid, '/formatters/xml')"
+                              />')
+                              </xsl:attribute>XML (<xsl:value-of select="if (standardName != '') then standardName else geonet:info/schema"/>)
+                            </fo:basic-link>
+                          </fo:inline>
+
+                        </xsl:with-param>
+                      </xsl:call-template>
+                    </fo:table-body>
+                  </fo:table>
+                </fo:block>
+              </fo:table-cell>
+            </fo:table-row>
+            <fo:table-row>
```

Comment:
I like it more displaying the thumbnail using the same format as other fields with 2 columns in the table: label, field, similar to the extent image.

But not really important and it's a matter of taste.

Reply:
> But not really important and it's a matter of taste.

I made changes based on a PDF example document made by designer... And do we really need a label to say that an image is an overview. Would be more relevant to add the overview label below the image maybe at some point.